requirements.txt

```diff
@@ -0,0 +1,16 @@
+tensorflow-gpu==1.15.5
+dash==1.19.0
+jupyter-dash==0.4.0
+dash_bootstrap_components==0.11.1
+dash_core_components==1.15.0
+dash_html_components==1.1.2
+progressbar2==3.53.1
+tables==3.6.1
+sqlalchemy==1.3.20
+openpyxl==3.0.6
+tabulate==0.8.7
+autopep8==1.5.4
+protobuf==3.14.0
+chembl_webresource_client==0.10.2
+dask_ml==1.8.0
+git+https://github.com/jrwnter/cddd.git
```

Comment:
This would be removed as well if we decide to remove the embeddings.

Reply:
same as above.